src/sdk/bucketspace_fs.js

```diff
@@ -485,12 +487,21 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
     // BUCKET TAGGING //
     ////////////////////
 
-    async put_bucket_tagging(params) {
+    /**
+     * @param {*} params 
+     * @param {nb.ObjectSDK} object_sdk 
+     */
+    async put_bucket_tagging(params, object_sdk) {
         try {
             const { name, tagging } = params;
             dbg.log0('BucketSpaceFS.put_bucket_tagging: Bucket name, tagging', name, tagging);
             const bucket = await this.config_fs.get_bucket_by_name(name);
-            bucket.tag = tagging;
+            const { ns } = await object_sdk.read_bucket_full_info(name);
+            const is_bucket_empty = await BucketSpaceFS.#_is_bucket_empty(name, null, ns, object_sdk);
```

Comment:
Why do you pass `null` as the params?

Reply:
Because I wanted to not provide and params in this case.

`undefined` vs `null` - this is quite opinionated. I prefer using `null` over `undefined` when I intentionally want to have a zero value. It helps me tell when the zero value was intentional and when the zero value is system generated.

I am aware that Guy disagrees with this opinion as `null` is an object.